packages/data-broker/src/services/dhis/DhisService.js

```diff
@@ -5,6 +5,7 @@
 
 import keyBy from 'lodash.keyby';
```

Comment:
@kael89 started looking at this with @IgorNadj, I suspect the `Event analytics require at least one organisation unit id` error doesn't get triggered because aggregator catches it first?
```
  async fetchAnalytics(codeInput, fetchOptions, aggregationOptions = {}) {
    const { organisationUnitCode, organisationUnitCodes } = fetchOptions;
    if (!organisationUnitCode && (!organisationUnitCodes || !organisationUnitCodes.length)) {
      // No organisation unit code, return empty response
      return {
        results: [],
        metadata: {
          dataElementCodeToName: {},
        },
        period: periodFromAnalytics([], fetchOptions),
      };
    }
```

Reply:
Yes, that makes sense! So an error will not get thrown, but the returned data will be always empty.

The next step would be to investigate existing functionality using `useDeprecatedApi: false` and confirm that it's broken - if it's not broken, then explain how it could work without org unit codes